runtime/codert_vm/xnathelp.m4

```diff
@@ -967,6 +967,25 @@ END_PROC(jitDecompileOnReturnJ)
 
 })	dnl ASM_J9VM_ENV_DATA64
 
+START_PROC(jitReadBarrier)
+ifdef({OMR_GC_CONCURRENT_SCAVENGER},{
+	SWITCH_TO_C_STACK
+	SAVE_C_VOLATILE_REGS
+	dnl currentThread->javaVM->memoryManagerFunctions->J9ReadBarrier(currentThread, (fj9object_t*)currentThread->floatTemp1);
+	mov _rax, uword ptr J9TR_VMThread_javaVM[_rbp]
```

Comment:
Since I'm being incredibly nitpicky - the rest of the code does not have space after comma.

Reply:
Thank-you for pointing it out. It has been updated.